source/simulator/assemblers/newton_stokes.cc

```diff
@@ -268,11 +268,18 @@ namespace aspect
       const unsigned int n_q_points    = scratch.finite_element_values.n_quadrature_points;
       const double derivative_scaling_factor = this->get_newton_handler().parameters.newton_derivative_scaling_factor;
 
-      const MaterialModel::AdditionalMaterialOutputsStokesRHS<dim>
-      *force = scratch.material_model_outputs.template get_additional_output<MaterialModel::AdditionalMaterialOutputsStokesRHS<dim> >();
 
-      const MaterialModel::ElasticOutputs<dim>
-      *elastic_outputs = scratch.material_model_outputs.template get_additional_output<MaterialModel::ElasticOutputs<dim> >();
+      const bool enable_additional_stokes_rhs = this->get_parameters().enable_additional_stokes_rhs;
+
+      const MaterialModel::AdditionalMaterialOutputsStokesRHS<dim> *force = nullptr;
+      if (enable_additional_stokes_rhs == true)
```

Comment:
```suggestion
      if (enable_additional_stokes_rhs)
```

Reply:
ok, will fix it. I wasn't sure any more what convention aspect uses :)